cmake/mpiexec.hera

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
maintaining a file for each platform is a ridiculous approach.

Reply:
Agree